sdk/metric/aggregator/exponential/mapping/logarithm/logarithm.go

```diff
@@ -169,3 +188,17 @@ func (l *logarithmMapping) LowerBoundary(index int32) (float64, error) {
 func (l *logarithmMapping) Scale() int32 {
 	return l.scale
 }
+
+// getSignificand returns the 52 bit (unsigned) significand as a
+// signed value.
+func getSignificand(value float64) int64 {
+	return int64(math.Float64bits(value)) & exponent.SignificandMask
```

Comment:
This is repeated from the `exponent` package and depends on a value from there.  Should it be exported and reused rather than reimplemented?

Reply:
Maybe an internal package?
I'm not sure how breaking it would be, but a bunch of constants in the exponent mapping package do not need to be exported, but they were exported so that ../logarithm could see them. Would you object if I moved everything inessential into an internal package, even if it meant breaking a user who had referred to `exponent.SignificandWidth` for example?

Reply:
These aren't stable packages, are they?  I'm fine with making breaking changes if they improve the interface.  I guess the question I'd have would be why an external user would be referring to that constant and whether that need would continue and force them to assume/duplicate it or is it eliminated by changes made here or that could be made.

Reply:
I agree. I'll update this PR to make things internal.

Reply:
Fixed. The two helper functions and all the internal constants are in an `./internal` package now.